openedx/core/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -238,6 +277,40 @@ def test_dop_public_client_access_token(self):
         data = json.loads(response.content)
         self.assertNotIn('refresh_token', data)
 
+    @ddt.data(dot_models.Application.GRANT_CLIENT_CREDENTIALS, dot_models.Application.GRANT_PASSWORD)
+    def test_jwt_access_token_scopes_and_filters(self, grant_type):
```

Comment:
Note: Not sure if it matters, but this functionality isn't tested when the application is Restricted, which is what we'll be doing in production.

Reply:
It does not matter. The fact that an application is restricted has no effect on the scope and filter claims that are added to the JWT which is what we are testing here.